.github/workflows/ci.yml

```
@@ -116,7 +116,9 @@ jobs:
         run: meson compile -C build
 
       - name: Check libvips
-        run: meson test -C build
+        run: |
+          meson test -C build \
+          || (cat build/meson-logs/testlog.txt && exit 1)
```

This can be simplified using the `--print-errorlogs` argument.
```suggestion
        run: meson test -C build --print-errorlogs
```

---

Oh nice, I didn't know about that.

---

... I swapped it back, `--print-errorlogs` only prints the last 100 lines and we need more than that in many cases.

---

> `--print-errorlogs` only prints the last 100 lines and we need more than that in many cases.

Ah, indeed. It seems that also passing `-v` would print the entire log.
https://github.com/mesonbuild/meson/blob/1.3.2/mesonbuild/mtest.py#L660-L661